stages/test/test_skopeo.py

```diff
@@ -33,3 +42,95 @@ def test_schema_validation_skopeo(stage_schema, test_data, expected_err):
     else:
         assert res.valid is False
         testutil.assert_jsonschema_error_contains(res, expected_err, expected_num_errs=1)
+
+
+@pytest.mark.skipif(os.getuid() != 0, reason="needs root")
+@pytest.mark.skipif(not has_executable("podman"), reason="no podman executable")
+@pytest.mark.parametrize("dest_type,local_output_path", [
+    ("dir", "/tmp/test-output-skopeo-dir"),
+    ("oci-archive", "/tmp/test-output-skopeo.tar"),
+    ("oci", "/tmp/test-output-skopeo-dir"),
+])
+def test_skopeo_dir(tmp_path, stage_module, dest_type, local_output_path):
```

Comment:
(super nitpick) maybe `test_skopeo_copy()` as it's more than jsut about "dir" ?

Reply:
...started with just `_dir` and added the parameterize later :-/
thanks, I'll change